helm/aws-load-balancer-controller/templates/webhook.yaml

```diff
@@ -142,7 +142,7 @@ data:
   tls.crt: {{ $tls.clientCert }}
   tls.key: {{ $tls.clientKey }}
 {{- else }}
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: cert-manager.io/v1
```

Comment:
Customers with old cert-manager setup will not be able to upgrade to the new load balancer controller chart since the old setup doesn't recognize v1 version. This will confuse the end users.

Is it possible to use `.Capabilities.APIVersions.Has` to use v1 only if support is available?

Reply:
It can be done as follows, but `--validate` is required if we want to use CRD apiversion for `.Capabilities.APIVersions.Has`.

```
helm template aws-load-balancer-controller ./aws-load-balancer-controller -f settings.yaml -n kube-system --validate
```


https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/2189/commits/fe800008ac1599b881b22c2640402aaf6f66dfcb